package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "floating-label-react",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
This shouldn't be 1.0, as it's not really a 1.0 product. If anything this should be 0.1.1. I haven't worked on this in a long time, so I'm not even sure it works anymore.

Reply:
All works fine, I added new commit with 0.1.1 version.